kernelci/rootfs.py

```diff
@@ -81,3 +93,86 @@ def upload(api, token, upload_path, input_dir):
             px = os.path.relpath(root, input_dir)
             artifacts[os.path.join(px, f)] = open(os.path.join(root, f), "rb")
     upload_files(api, token, upload_path, artifacts)
+
+
+def validate(configs):
```

Comment:
Please add a docstring to all public functions or methods.

Reply:
I wonder if this should rather be in `kernelci.config.rootfs` as it's really all about the config data?

Reply:
docstrings added, moved config related functions to kernelci.config.rootfs